pytorch/training/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -373,6 +373,13 @@ ENV SAGEMAKER_TRAINING_MODULE=sagemaker_pytorch_container.training:main
 ARG SMD_DATA_PARALLEL_URL
 ARG SMD_MODEL_PARALLEL_URL
 
+# EFA checker binaries
+ARG EFA_CHECKER_DLC_SINGLE_NODE_URL=https://smdataparallel.s3.amazonaws.com/efa-health-checher/dlc-pytorch-2.0.1/efa_checker_single_node_dlc_with_mpi
```

Comment:
Can we use [this](https://github.com/aws/deep-learning-containers/pull/3293/files#diff-bc0b77b02ee5079e588e1fd8714585bee92e4159c585a980f2f576beffbc6228R6) ARG here in the name of the bucket. So, `dlc-pytorch-2.0.1` should look like `"dlc-pytorch-{PYTORCH_VERSION_SM}"`

Reply:
I think EFA checker binary is manually put into https://smdataparallel.s3.amazonaws.com/efa-health-checher/dlc-pytorch-2.0.1. In other words, when there is pytorch-2.1, EFA checker binary will not be automatically generated at https://smdataparallel.s3.amazonaws.com/efa-health-checher/dlc-pytorch-2.1